api/ooniapi/aggregation.py

```diff
@@ -203,6 +208,7 @@ def get_aggregated() -> Response:
         inp = param_input_or_none() or ""
         probe_asn_s = param_asn_m()
         probe_cc_s = param_probe_cc_m()
+        ooni_run_link_ids_raw = param("ooni_run_link_id")
```

Comment:
Variable name still contains `s` at the end

Reply:
Updated